### DISCUSSION TITLE
fci_tc_bi_ortho incorrect calculation

### DISCUSSION THREAD

**Discussion by kevrap121** [QuantumPackage/qp2#340] [Q&A] [answered]
Hello @scemama ! I am trying to run a tc fci for Li2 to benchmark and get familiar with the Transcorrelation tools of QuantumPackage, but I'm getting energies that are below the energy minimum. I know this should be the case for the tc_scf, but I'm not sure why it is happening when I run fci_tc_bi_ortho. I attached a tarbell of the coordinates and script I am using, and I was wondering if you knows why this is happening. I also included results for when I set read_wf to false (start with a ROHF-like Slater determinant guess) and for when I set read_wf to true (start with the EZFIO wave function).
[qp2-2.2.2_archive.tar.gz](https://github.com/user-attachments/files/16239473/qp2-2.2.2_archive.tar.gz)

**Comment by eginer** [comment, marked as answer]
Hello @kevrap121 ! 
I had a look at your script, and I have fixed it. 
Roughly speaking, the TC energy strongly depends on the shape of the Jastrow, and the default parameter you used are not the best (you could not know :) 
I attach an autonomous script to compute the energy of Li2 in the aug-cc-pVDZ (it runs very fast) with all detailed explained and some numbers to compare with. 
WARNING: a ) you have to put in the script the QP_ROOT variable (where your QP2 is installed). 
                    b ) I fixed some bugs in the QP2 related to TC (small stuffs) so I strongly suggest you to git pull & ninja clean & ninja 
                         The commit code is "4af118c4e62fd99413f0917efab7b16cf66529fd" 
I will not be available before September to potentially reply to further questions, but I will answer when I come back. 
All the best,

1. Emmanuel 
[example_tc.zip](https://github.com/user-attachments/files/16253989/example_tc.zip)

**Comment by kevrap121** [comment]
Hello @eginer!

Thank you so much for this feedback. It is super helpful and I was able to get some cool results for a singlet and triplet carbon dimer using the parameter source you recommended. I'm curious to use the three-body Jastrow tools in the plugins, but I can't seem to find documentation for how to use them / set parameters. Could you point me in the right direction?

Best,
Kevin
